src/app/screens/Publishers/index.tsx

```diff
@@ -31,7 +35,14 @@ function Publishers() {
             },
           };
         }
-        return allowance;
+        if (
+          blocklistResponse.blocklist.find(
+            (item) => item.host === allowance.host
+          )
+        ) {
+          retobj.blocked = true;
```

Comment:
for simplicity reasons I think we could remove this here. 

A user can only add a page to the blocklist on the first `enable` call. In that case no `allowance` is saved. 
And because we show an unblock button directly on the popup home screen I think this is enough.

what do you think? 


in the future I also think of a permission system with very granular permissions per host. 
Then I think we have to rethink where/how we show this anyway.

Reply:
remove what exactly? I don't think this can be removed - this is where we are looping through the blocklist and finding blocked host. 
agree that this will need to be rewritten at some point.

Reply:
yes, but when is a host blocked? isn't it normally the case that a host is blocked OR it is in the allowance list? 
when a host gets blocked it is NOT added to the allowance list. 

or what am I missing?

Reply:
I think you're right. this may be left over from when I was thinking that blocklist would go into a list in publishers screen. But we're not doing that at the moment. Removed.